addons/addon-base-raas/packages/base-raas-services/lib/environment/service-catalog/environment-sc-connection-service.js

```diff
@@ -190,7 +199,16 @@ class EnvironmentScConnectionService extends Service {
     const credentials = `${username}\n${password}`;
     const publicKey = await this.getRstudioPublicKey(requestContext, instanceId, id);
     const [exponent, modulus] = publicKey.split(':', 2);
-    const params = { v: rstudioEncryptor.encrypt(credentials, exponent, modulus) };
+    const exponentBuffer = Buffer.from(exponent, 'hex');
```

Comment:
I see this section of code is used twice. Once here and another time in `environment-url-service.js`. I might be helpful to create a util method out of this and put it in `addons/addon-base-raas/packages/base-raas-services/lib/helpers/util.js` to DRY up the code.

Reply:
We could, but the code in `environment-url-service` is deprecated since it was being used for built-in workspaces.

Reply:
Not moving code to util, as discussed over chat.